### ISSUE TITLE
missing underscore in image-filenames at static image-layers

### ISSUE THREAD

**Issue by swen100** [tomchadwin/qgis2web#551]
After exporting static georeferenced images, the filenames of the images have an underscore inside the layers-definitions (both in OL3 and Leaflet), like imagefile_1, imagefile_2, ... Their filenames get an incremented number separated by underscore. But the filenames of the copied images doesn't contain this underscore, so they won't get displayed.

Example:
copied image-file: **58058_B22_03Ae1.png**
generated JS-Code:

```javascript
var lyr_58058_B22_03Ae1 = new ol.layer.Image({
    opacity: 1,
    title: "58058_B22_03Ae",
    source: new ol.source.ImageStatic({
        url: "./layers/58058_B22_03Ae_1.png",
        attributions: [new ol.Attribution({html: '<a href=""></a>'})],
        projection: 'EPSG:3857',
        alwaysInRange: true,
        //imageSize: [2952, 3029],
        imageExtent: [1112638.280858, 7246675.730827, 1112856.181189, 7246893.494201]
    })
});
```
58058_B22_03Ae1.png <> 58058_B22_03Ae_1.png

**Comment by tomchadwin** [comment]
Which version of qgis2web are you using? It works for me here.

**Comment by swen100** [comment]
Hi Tom,
I tried it out with QGIS 2.14.19 (on two different computers, 32bit and 64bit) and with newest qgis2web 2.28.0.
Btw, the images were grouped togther inside a layer-group in QGIS.

**Comment by tomchadwin** [comment]
Could you upload a project and raster file which exhibit this behaviour?

**Comment by geoglis** [comment]
It works nearly, raster are no more reprojected, fine. But my geotif files are stored in the layers directory with file extension png, but they are still tifs. There is an artificial renaming in the code, I think.
Bytheway: They are bigger now, my uncompressed geotifs were 71 MB, now they are 95 MB, both same size and uncrompressed and only 71 MB in RAM, says InrfanView.

**Comment by geoglis** [comment]
With the new master made for #552 the filenames are now correctly with underscore "_n", there seems to be an old code part in the actual plugin repository version of qgis2web.

**Comment by swen100** [comment]
solved in the current master

**Comment by tomchadwin** [comment]
Sorry, I thought that fix was in the latest version. Glad it's fixed, and I'll fry to get a new version out in the next couple r of days.

**Comment by tomchadwin** [comment]
Released as 2.29.0 and pushed to QGIS plugins repo.
